src/components/orders/new-order.component.ts

```diff
@@ -404,7 +461,7 @@ export class NewOrderComponent implements OnInit {
                     this.ShowAlert('order Saved', 1);
                     setTimeout(() => {
                         this.store.dispatch(new Navigate(['/orders']))
-                    }, 3500)
```

Comment:
Eddy requested to show the message for a longer time

Reply:
in additon, the atribute top on the fixed styles  have to be fixed on products, an inventory item: esampel top: 7.3rem;